### ISSUE TITLE
add XYZ coords

### ISSUE THREAD

**Issue by JulianKlug** [MonsieurWave/stroke-predict#15]
High risk of overfitting as dataset might be too small 
--> xgboost will learn coordinates of lesioned sections in train sets
--> bad perf in test set

(probably)

Report testing performance to check for overfit
